src/utils/useLoadingColors.ts

```diff
@@ -0,0 +1,34 @@
+import { ThemeColors } from '@theme/types';
+import color from 'color';
+import { useAppSettings } from '@hooks/persisted';
+import { interpolateColor } from 'react-native-reanimated';
+
+const useLoadingColors = (theme: ThemeColors) => {
```

Comment:
renamed to use cus of the useAppSettings inside

Reply:
maybe lint gives problems because Traitor latest commit bypassed lint verification.